tests/templates/kuttl/upgrade/00-assert.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: install-zk
```

Comment:
As i learned recently this doesnt do anything: https://kuttl.dev/docs/testing/reference.html#testassert

Reply:
You meant the `name` doesn't do anything?